utils/build-script

```diff
@@ -353,6 +353,21 @@ class BuildScriptInvocation(object):
             args.skip_test_tvos = True
             args.skip_test_watchos = True
 
+        # --build-swift-stdlib is an alias for --build-swift-dynamic-stdlib=1
+        # and --build-swift-static-stdlib=0, which takes precedence over those
+        # options.
+        if args.build_swift_stdlib:
+            args.build_swift_dynamic_stdlib = True
+            args.build_swift_static_stdlib = False
```

Comment:
There's a way, but we'd need to stop using the `optional_bool` action.  Since this is a short-lived compatibility aid, I think inspecting `argv` is acceptable.